addons/survey/static/src/js/survey_result.js

```diff
@@ -325,20 +332,33 @@ publicWidget.registry.SurveyResultChart = publicWidget.Widget.extend({
                 datasets: datasets
             },
             options: {
-                title: {
-                    display: true,
-                    text: _t("Performance by Section"),
-                },
-                legend: {
-                    display: true,
+                plugins : {
+                    title: {
+                        display: true,
+                        text: _t("Performance by Section"),
+                    },
+                    legend: {
+                        display: true,
+                    },
+                    tooltip: {
+                        callbacks: {
+                            label: function (tooltipItem, data) {
```

Comment:
outdated API.
The new API is `function (tooltipItem)`
https://www.chartjs.org/docs/latest/api/interfaces/TooltipCallbacks.html#label

Same issue in the enterprise branch in `hr_payroll` (doesn't seem to have a PR yet)

https://github.com/odoo-dev/enterprise/blob/e5d4b0d5634268ac8e4c6af1d392eaa403c1b3be/hr_payroll/static/src/components/dashboard/payroll_stats/payroll_stats.js#L249